dora/core/client/fs/src/main/java/alluxio/client/file/ufs/UfsFileInStream.java

```diff
@@ -184,4 +195,16 @@ private void updateStreamIfNeeded() {
     }
     mUfsInStream = Optional.of(ufsInStream);
   }
+
+  /**
+   * Class that contains metrics about FileOutStream.
+   */
+  private static final class Metrics {
+    // Note that only counter can be added here.
```

Comment:
how come this restriction?

Reply:
I don't know the reason of this restriction.
This is following existing code in other places.

Reply:
removed the comment.